latch_cli/main.py

```diff
@@ -41,10 +41,12 @@ def main():
     latest_ver = parse_version(get_latest_package_version())
     if local_ver < latest_ver:
         click.secho(
-            textwrap.dedent(f"""
+            textwrap.dedent(
```

Comment:
update ur version of black to `23.7.0` and add the `--preview` flag so we don't have reformatting happening